prusa/connect/printer/camera.py

```diff
@@ -12,15 +12,30 @@ class CameraMgr:
     """Camera Manager representation object."""
     snapshot_queue: "Queue[Snapshot]"
 
-    def __init__(self, conn, server):
+    def __init__(self, conn, queue):
         """
         :conn: Current RetryingSession connection
         :server Connect server URL
         """
         self.conn = conn
-        self.server = server
+        self.server = None
```

Comment:
Really need we server attribute?

Reply:
As we discussed, it's there because of the Snapshot. Here I've just changed the implicit value.